$MirOS: ports/www/firesomething/patches/patch-security_coreconf_config_mk,v 1.1.7.1 2005/03/18 15:59:34 tg Exp $
--- security/coreconf/config.mk.orig	Sun Apr 25 17:01:55 2004
+++ security/coreconf/config.mk	Thu Dec  8 20:42:53 2005
@@ -63,7 +63,7 @@ endif
 #######################################################################
 
 TARGET_OSES = FreeBSD BSD_OS NetBSD OpenUNIX OS2 QNX Darwin BeOS OpenBSD \
-              OpenVMS AIX
+              MirBSD OpenVMS AIX
 
 ifeq (,$(filter-out $(TARGET_OSES),$(OS_TARGET)))
 include $(CORE_DEPTH)/coreconf/$(OS_TARGET).mk
